README.md

```diff
@@ -47,6 +47,10 @@ The JSON object will look like this:
 
 This will allow our users to be able to get data about the object, and search for objects containing specific information.
 
+##Necessary Precautions For This Project
+
+* Within the Maven tab, install the Maven Dependencies (Source & Documentation)
+
```

Comment:
Although this may have just been negligence on my part and forgetting an important step, I added this to your ReadMe as the project did not compile until I did this. I would say for an outsider, this would be important to include.